cmd/kyma/alpha/create/module/opts.go

```diff
@@ -5,16 +5,18 @@ import (
 	"os"
 	"path/filepath"
 
+	"regexp"
+
 	"github.com/kyma-project/cli/internal/cli"
 	"github.com/pkg/errors"
-	"regexp"
 )
 
 // Options defines available options for the create module command
 type Options struct {
 	*cli.Options
 
 	Name                 string
+	NameMappingMode      string
```

Comment:
I've initially have this, but (most) other validations happen in the `validate()` function inside `pkg/module/def.go`. So I've moved it there. The validation inside module is safer because it's closer to the business logic, and we have more natural access to some constants - like the ones imported already from gardener's libs. Having double-validation here wasn't worth it in my opinion. But if you think we should have it, I'll add it again.

Reply:
sounds reasonable to me, lets leave it